tests/console/zypper_lifecycle.pm

```diff
@@ -37,7 +37,8 @@ our $date_re = qr/[0-9]{4}-[0-9]{2}-[0-9]{2}/;
 
 sub lifecycle_output_check {
     my $output = shift;
-    if (is_sle('=15-sp1') && $output =~ /Python 2 Module.*2021-07-31/) {
+
```

Comment:
please remove extra line.
We new code we can separate logically parts, but for existing code shared I don't recommend it, just change the minimum number of lines and characters :)

Reply:
Done.